test/support/schemas/hyper_schema.json

```diff
@@ -0,0 +1,93 @@
+{
+  "$schema": "http://json-schema.org/draft-04/hyper-schema",
```

Comment:
are you using something to generate these?

might also be nice to add  some JSON API stuff even like in https://github.com/rails-api/active_model_serializers/pull/1301#discussion_r44242510 (which should probably be merged, I guess)